<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="OSO-DTP-057" name="Employees Attendance Calendar" tests="3" failures="1" timestamp="3 Oct 2017 05:33:52 GMT" time="71.677" errors="0">
  <testcase name="Login" time="1.257" classname="ESS.LeaveandAttendance.LeaveApplication.TC015_EmployeeAttendanceCalender_Verify"/>
  <testcase name="TargetPage" time="39.132" classname="ESS.LeaveandAttendance.LeaveApplication.TC015_EmployeeAttendanceCalender_Verify"/>
  <testcase name="VerifyAttendanceApproveORNot" time="0.177" classname="ESS.LeaveandAttendance.LeaveApplication.TC015_EmployeeAttendanceCalender_Verify">
    <failure type="org.testng.TestNGException" message="
The data provider is trying to pass 2 parameters but the method ESS.LeaveandAttendance.LeaveApplication.TC015_EmployeeAttendanceCalender_Verify#VerifyAttendanceApproveORNot takes 3 and TestNG is unable in inject a suitable object">
      <![CDATA[org.testng.TestNGException: 
The data provider is trying to pass 2 parameters but the method ESS.LeaveandAttendance.LeaveApplication.TC015_EmployeeAttendanceCalender_Verify#VerifyAttendanceApproveORNot takes 3 and TestNG is unable in inject a suitable object
Caused by: java.lang.ArrayIndexOutOfBoundsException: 2
	... 17 more
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- VerifyAttendanceApproveORNot -->
</testsuite> <!-- Employees Attendance Calendar -->
